### PR TITLE
Client policies(conditions): add correct form fields for all condition types

### PR DESCRIPTION
**Pull request by jenny-s51**
## Motivation

https://issues.redhat.com/browse/CLUX-299

## Brief Description

Adds functionality for the relevant configuration attributes for all policy condition types. 

## Verification Steps

1. Go to Client policies -> policies tab
2. Select a client policy or create a new one from the empty state. 
3. Click "Add a condition"
4. Verify that the correct fields and help text are showing for all condition types.
3. Add a condition 
4. Verify that you are able to save and submit successfully, and that the data is written correctly to the server

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [x] Help has been implemented
- [x] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

## Additional Notes

Builds on #1393 and #1357 - will rebase once merged to simplify review process

### CODE REVIEW

**Comment by jenny-s51** [comment]
@ssilvert fixed!

File: src/client-scopes/add/components/MultivaluedListComponent.tsx [hunk 1]
```diff
@@ -18,15 +19,29 @@ export const MultivaluedListComponent = ({
...
   options,
 }: ComponentProps) => {
-  const { t } = useTranslation("client-scopes");
+  const { t } = useTranslation("realm-settings-help");
   const { control } = useFormContext();
   const [open, setOpen] = useState(false);
+  const [selectedItems, setSelectedItems] = useState<string[]>([defaultValue]);
+
+  const convertToString = (s: string) =>
+    camelCase(s.replaceAll(".", "-").replaceAll("-", " "));
```
Comment (jonkoops) [inline src/client-scopes/add/components/MultivaluedListComponent.tsx:28]: Looking for this code in the project I can find it exactly 6 times, perhaps it is starting to make sense to split this out in a utility function.

**Review by jonkoops — APPROVED** [review]

**Comment by ssilvert** [comment]
> @ssilvert fixed!

@jenny-s51 Now I can no longer create a client policy.

**Comment by ssilvert** [comment]
> > @ssilvert fixed!
> 
> @jenny-s51 Now I can no longer create a client policy.

Hmm.  This appears to be something that is broken in master.

**Comment by jenny-s51** [comment]
@ssilvert yes it looks like the route was changed in Agi's PR which was merged earlier today ... https://github.com/keycloak/keycloak-admin-ui/pull/1450/files#diff-e9bd4be211c473f1940cf2eb43b009ca1a216d07632c521f494d6a2c4dba6171R12

will revert @agagancarczyk 's change and push again

**Review by ssilvert — APPROVED** [review]
Looking good.

**Pull request merged**